src/Uno.UWP/UI/Core/CoreWindow.wasm.cs

```diff
@@ -1,26 +0,0 @@
-﻿using Uno.Foundation;
-
-using NativeMethods = __Windows.UI.Core.CoreWindow.NativeMethods;
-
-namespace Windows.UI.Core;
-
-public partial class CoreWindow
-{
-	private CoreCursor _pointerCursor = new CoreCursor(CoreCursorType.Arrow, 0);
-
-	public global::Windows.UI.Core.CoreCursor PointerCursor
-	{
-		get => _pointerCursor;
-		set
-		{
-			_pointerCursor = value;
-			Internal_SetPointerCursor();
-		}
-
-	}
```

Comment:
Why is it removed?

Reply:
Now using the version shared for targets that uses managed pointers https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/UI/Core/CoreWindow.Pointers.Managed.cs